VRCFaceTracking/Params/Expressions/Legacy/Lip/UnifiedSRanMapper.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace VRCFaceTracking.Params.Lip
+{
+    class UnifiedSRanMapper
+    {
+        private static Dictionary<SRanipal_LipShape_v2, Func<UnifiedTrackingData, float>> ShapeMap = new Dictionary<SRanipal_LipShape_v2, Func<UnifiedTrackingData, float>>
+        {
+            { SRanipal_LipShape_v2.JawRight, exp => exp.Shapes[(int)UnifiedExpressions.JawRight].AdjustedWeight },
+            { SRanipal_LipShape_v2.JawLeft, exp => exp.Shapes[(int)UnifiedExpressions.JawLeft].AdjustedWeight },
+            { SRanipal_LipShape_v2.JawForward, exp => exp.Shapes[(int)UnifiedExpressions.JawForward].AdjustedWeight },
+            { SRanipal_LipShape_v2.JawOpen, exp => exp.Shapes[(int)UnifiedExpressions.JawOpen].AdjustedWeight - exp.Shapes[(int)UnifiedExpressions.MouthClosed].AdjustedWeight },
```

Comment:
Most likely will need to look at JawOpen and MouthApeShape in testing as this can be tricky to make work as good as native SRanipal mouthApeShape

Reply:
It seems to work as intended from my own testing. It should be mathematically consistent as far as removing the Ape part directly from SRanipal as MouthClose is using the same value as MouthApe from SRanipal. This also forces any new tracking interfaces to behave like SRanipal would when using the old parameters. Sending the data to JawOpen just happens to have both Sranipal JawOpen and SRanipal MouthApe baked into the value as I'm considering them to both open the jaw from the SRanipal tracking interface.

We could consider making a new output parameter named MouthApeShape for v2 that acts like SRanipal Ape but I think we should try to avoid losing fidelity in the new standard's combined params.